src/components/charts/shared/navDrawer/index.jsx

```diff
@@ -55,44 +54,38 @@ class NavDrawer extends React.PureComponent {
 
     return (
       <>
-        {showNavDrawer && (
-          <Card className={cardContainerClasses}>
-            <NavDrawerToolbar
-              menuIsOpen={menuIsOpen}
-              toggleMenu={this.menuToggler}
-              title={toolbarTitle}
-              actions={toolbarActions}
-              interactableToolbar={interactableToolbar}
-              toolbarStyles={toolbarStyles}
-            />
-            <NavigationDrawer
-              navItems={navItems}
-              visible={menuIsOpen}
-              onVisibilityChange={this.handleVisibilityChange}
-              position="left"
-              className={navClasses}
-              drawerClassName={drawerClasses}
-              mobileDrawerType={NavigationDrawer.DrawerTypes.TEMPORARY_MINI}
-              tabletDrawerType={NavigationDrawer.DrawerTypes.PERSISTENT_MINI}
-              desktopDrawerType={NavigationDrawer.DrawerTypes.PERSISTENT_MINI}
-              overlay={false}
-              contentClassName={contentClasses}
-            >
-              {children}
-            </NavigationDrawer>
-          </Card>
-        )}
+        <Card className={cardContainerClasses}>
+          <NavDrawerToolbar
+            menuIsOpen={menuIsOpen}
+            toggleMenu={this.menuToggler}
+            title={toolbarTitle}
+            actions={toolbarActions}
+            interactableToolbar={interactableToolbar}
+            toolbarStyles={toolbarStyles}
+          />
+          <NavigationDrawer
+            navItems={navItems}
+            visible={menuIsOpen}
+            onVisibilityChange={this.handleVisibilityChange}
+            position="left"
+            className={navClasses}
+            drawerClassName={drawerClasses}
+            mobileDrawerType={NavigationDrawer.DrawerTypes.TEMPORARY_MINI}
+            tabletDrawerType={NavigationDrawer.DrawerTypes.PERSISTENT_MINI}
+            desktopDrawerType={NavigationDrawer.DrawerTypes.PERSISTENT_MINI}
+            overlay={false}
+            contentClassName={contentClasses}
+          >
+            {children}
+          </NavigationDrawer>
+        </Card>
 
-        {!showNavDrawer && children}
+        {children}
```

Comment:
Should take this line out altogether.  As is every page with a NavDrawer has duplicated content below the NavDrawer